### ISSUE TITLE
[BUG] cfn-guard validate doesn't produce a parseable json/yaml output in 'custom_message' field

### ISSUE THREAD

**Issue by omartrigui** [aws-cloudformation/cloudformation-guard#389]
**Describe the bug**

The `cfn-guard validate` command produces an invalid JSON/YAML string in the `Clause.Unary.messages.custom_message` field. The resulting string contains several `;` characters instead of `\n` that cause parsing failures.

**To Reproduce**

##### cf-rds.yaml
```yaml
AWSTemplateFormatVersion: "2010-09-09"
Description: "Rocket league DB"
Resources:
  DBInstance:
    Type: "AWS::RDS::DBInstance"
    Properties:
      DBInstanceIdentifier: "MyRDSInstance"
      Engine: "mysql"
      DBInstanceClass: "db.t2.micro"
      AllocatedStorage: "20"
      MasterUsername: "dbadmin"
      MasterUserPassword: "MyStrongPassword123"
      AllowMajorVersionUpgrade: true
      VPCSecurityGroups:
        - !Ref RDSSecurityGroup
  RDSSecurityGroup:
    Type: "AWS::EC2::SecurityGroup"
    Properties:
      GroupDescription: "RDS Security Group"
      SecurityGroupIngress:
        - CidrIp: "0.0.0.0/0"
          IpProtocol: "tcp"
          FromPort: "3306"
          ToPort: "3306"
```
##### rds.guard
```
# RDS Rules
let aws_rds = Resources.*[ Type == 'AWS::RDS::DBInstance' or Type == 'AWS::RDS::DBCluster' ]

rule enforce_aws_rds_is_encrypted when %aws_rds !empty {
    %aws_rds {
        Properties {
            StorageEncrypted exists
            <<
                {
                    "Id": "ENF-AWS-RDS-003",
                    "Message": "RDS storage is not encrypted.",
                    "ReferenceURL": "https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbinstance.html#cfn-rds-dbinstance-storageencrypted"
                }
            >>
        }
    }
}
```

##### command to reproduce
```bash
$ cfn-guard -V
cfn-guard 3.0.0

$ cat cf-rds.yaml | cfn-guard validate -r rds.guard -o json --show-summary none | jq '.not_compliant[0].Rule.checks[0].Clause.Unary.messages.custom_message'
";                {;                    \"Id\": \"ENF-AWS-RDS-003\",;                    \"Message\": \"RDS storage is not encrypted.\",;                    \"ReferenceURL\": \"https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-rds-dbinstance.html#cfn-rds-dbinstance-storageencrypted\";                };            "
```
**Expected behavior**

`cfn-guard` displays a parseable JSON/YAML  `custom_message` data.

**Operating System:**
Ubuntu

**OS Version**
All versions

**Additional context**
Note that cfn-guard `2.0.3` generates a parseable `message` output

**Comment by joshfried-aws** [comment]
Hi @omartrigui thank you for bringing this issue to our attention. I was able to reproduce this issue. We will continue to track this issue and update it when a fix is out.

**Comment by joshfried-aws** [comment]
Hi @omartrigui I merged the fix for this in #396. Thanks again for bringing this to our attention. I am going to go ahead and close out this issue for now. Feel free to reopen if need be.
